features/font-family-math.yml

```
@@ -0,0 +1,5 @@
+name: Math font family
+description: The `font-family:\ math` CSS declaration uses the browser default font face for displaying mathematical expressions.
```

FYI, these slashes show up literally

```suggestion
description: The `font-family: math` CSS declaration uses the browser default font face for displaying mathematical expressions.
```

You can "preview" the HTML for these with `npx tsx -p 'import { features } from "./index.ts"; Object.values(features).forEach(f => console.l
og(f.description_html));' | grep 'system-ui'`

---

Removing it breaks YAML, and gives the error-

`YAMLParseError: Nested mappings are not allowed in compact mappings at line 2, column 14:`

I'm able to fix it by wrapping in double quotes.